StripeConnect/StripeConnectTests/Internal/Webview/MessageSenders/LogoutSenderTests.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LogoutSenderTests.swift
+//  StripeConnectTests
+//
+//  Created by Mel Ludowise on 9/24/24.
+//
+
+@testable import StripeConnect
+import XCTest
+
+class LogoutSenderTests: ScriptWebTestBase {
+    func testSendMessage() throws {
+        try validateMessageSent(sender: LogoutSender())
+    }
+
+    func testSenderSignature() {
+        XCTAssertEqual(
+            LogoutSender().javascriptMessage,
+            """
+            window.logout({});
```

Comment:
Ideally this would be empty args, but there's no easy way to do that with the MessageSender. I tested that sending extra arguments to this function doesn't seem to have any negative effect in JS.